sheet_to_triples/tests/test_rdf.py

```diff
@@ -0,0 +1,242 @@
+# Copyright 2021 Visual Meaning Ltd
+# This is free software licensed as GPL-3.0-or-later - see COPYING for terms.
+
+"""Unittests for the rdf.py module of sheet-to-triples."""
+
+import unittest
+
+import rdflib.term
+
+from .. import rdf
+
+
+class TestRDF(unittest.TestCase):
+
+    def test_from_qname(self):
+        term = rdf.from_qname('owl:test')
+        self.assertIsInstance(term, rdflib.term.URIRef)
+        self.assertEqual(str(term), 'http://www.w3.org/2002/07/owl#test')
+
+    def test_from_qname_prefix_vm(self):
+        term = rdf.from_qname('vm:test')
+        self.assertIsInstance(term, rdflib.term.URIRef)
+        self.assertEqual(str(term), 'http://visual-meaning.com/rdf/test')
+
+    def test_from_qname_prefix_http(self):
+        term = rdf.from_qname('http://test.test')
+        self.assertIsInstance(term, rdflib.term.URIRef)
+        self.assertEqual(str(term), 'http://test.test')
+
+    def test_from_identifier_already_ident(self):
+        term = rdflib.term.URIRef('test')
+        self.assertEqual(
+            rdf.from_identifier(term),
+            term
+        )
+
+    def test_from_identifier_sequencepath(self):
+        term = rdf.from_identifier('vm:a / vm:b')
+        expected = [
+            'http://visual-meaning.com/rdf/a',
+            'http://visual-meaning.com/rdf/b',
+        ]
+        self.assertIsInstance(term, rdflib.paths.SequencePath)
+        self.assertEqual(
+            [str(p) for p in term.args],
+            expected
+        )
```

Comment:
I think this block would be fine as:
```python
expected = rdflib.paths.SequencePath(
    'http://visual-meaning.com/rdf/a',
    'http://visual-meaning.com/rdf/b',
)
self.assertEqual(term, expected)
````
And trust the equality operation on the library type rather than deconstructing it.